yt/frontends/ramses/tests/test_outputs.py

```diff
@@ -244,7 +244,7 @@ def test_ramses_sink():
     ad = ds.all_data()
 
     for field in expected_fields:
-        assert ("sink", "field") not in ds.field_list
```

Comment:
I think this test was bug and this is an attempt to fix it

Reply:
I think so too

Reply:
Happens to the best of us 🥇